data-processing/scripts/memo/convert_memo_to_jsonlgz.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import json
+import gzip
+import os
+from tqdm import tqdm
+import datetime
+import nltk
+from nltk.tokenize import word_tokenize
+
+'''
+.txt (text) & .csv(metadata) -> json.gz:
+
+{
+    "id": "...",             # MANDATORY: source-specific identifier
+    "text": "foo",           # MANDATORY: textual content of the document
+    "source": "...",         # MANDATORY: source of the data, such as peS2o, common-crawl, etc.
+    "added": "...",          # OPTIONAL: timestamp we acquired this data (time file was created), specified as YYYY-MM-DDTHH:MM:SS.TIMEZONE e.g 2021-04-13T12:52:46.000Z
+    "created": "..."         # OPTIONAL: timestamp when orig document was created (best-guess if not available), should be specified as a range; "YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE"
+    "metadata": {            # OPTIONAL: source-specific metadata
+										"sub-source": "...", # OPTIONAL: E.g. "newspaper_ocr"
+										...
+								}        
+}
+'''
```

Comment:
```suggestion
'''
.txt (text) & .csv(metadata) -> json.gz:

{
    "id": "...",             # MANDATORY: source-specific identifier
    "text": "foo",           # MANDATORY: textual content of the document
    "source": "...",         # MANDATORY: source of the data, such as peS2o, common-crawl, etc.
    "added": "...",          # OPTIONAL: timestamp we acquired this data (time file was created), specified as YYYY-MM-DDTHH:MM:SS.TIMEZONE e.g 2021-04-13T12:52:46.000Z
    "created": "..."         # OPTIONAL: timestamp when orig document was created (best-guess if not available), should be specified as a range; "YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE"
    "metadata": {            # OPTIONAL: source-specific metadata
										"sub-source": "...", # OPTIONAL: E.g. "newspaper_ocr"
										...
								}        
}
'''


import pandas as pd
import json
import gzip
import os
from tqdm import tqdm
import datetime
import nltk
from nltk.tokenize import word_tokenize
```

assuming it is a docstring I would also write it out a bit. I.e. "This script converts ..."